modules/nf-core/bcftools/pluginimputeinfo/environment.yml

```diff
@@ -0,0 +1,7 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - "bioconda::bcftools=1.20"
```

Comment:
Does this come with the plugin? When I tried to install ityself there was no plugin in the package...

Reply:
It should, here is what I get running on gitpod
```
(base) gitpod /workspace/modules (bcftools/pluginimputeinfo) $ conda env create --name bcftools --file /workspace/modules/modules/nf-core/bcftools/pluginimputeinfo
/environment.yml --quiet
Channels:
 - conda-forge
 - bioconda
 - defaults
Platform: linux-64
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... done
Preparing transaction: ...working... done
Verifying transaction: ...working... WARNING conda.core.path_actions:verify(1050): Unable to create environments file. Path not writable.
  environment location: /home/gitpod/.conda/environments.txt

done
Executing transaction: ...working... WARNING conda.core.envs_manager:register_env(87): Unable to register environment. Path not writable or missing.
  environment location: /opt/conda/envs/bcftools
  registry file: /home/gitpod/.conda/environments.txt
done
(base) gitpod /workspace/modules (bcftools/pluginimputeinfo) $ conda activate bcftools
(bcftools) gitpod /workspace/modules (bcftools/pluginimputeinfo) $ bcftools plugin impute-info --help

About: Add imputation information metrics to the INFO field based
       on selected FORMAT tags. Only the IMPUTE2 INFO metric from
       FORMAT/GP tags is currently available.
Usage: bcftools +impute-info [General Options] -- [Plugin Options]
Options:
   run "bcftools plugin" for a list of common options

Example:
   bcftools +impute-info in.vcf
```

Reply:
Strange then 😅 but ok! Must be just my set up!